internal/controller/vrg_volrep.go

```diff
@@ -32,6 +32,21 @@ const (
 	defaultVRCAnnotationKey = "replication.storage.openshift.io/is-default-class"
 )
 
+//nolint:gosec
+const (
+	// secretRef keys
+	controllerPublishSecretName      = "csi.storage.k8s.io/controller-publish-secret-name"
```

Comment:
What about the provisioner secrets that are present in pvc annotation? Want we update that as well?

Reply:
Looks like am late 😕

Reply:
Provisioner secrets are in PV annotations? I assume these are needed for Delete operations?

If so @rakeshgm add a new PR to update that as well.

Reply:
```yaml
apiVersion: v1
items:
- apiVersion: v1
  kind: PersistentVolume
  metadata:
    annotations:
      pv.kubernetes.io/provisioned-by: rook-ceph.rbd.csi.ceph.com
      volume.kubernetes.io/provisioner-deletion-secret-name: rook-csi-rbd-provisioner
      volume.kubernetes.io/provisioner-deletion-secret-namespace: rook-ceph
    creationTimestamp: "2024-11-05T09:51:09Z"
    finalizers:
```
Yes in the pv annotation.

Reply:
Added a task in #1616